app/scripts/lib/app-start.js

```diff
@@ -215,6 +224,15 @@ function (
       this._metrics.init();
     },
 
+    initializeIframeChannel: function () {
+      if (this._isIframe()) {
```

Comment:
I suppose the RP could still listen on this channel even if it's not being used to drive the auth flow.